File: core/src/main/java/vn/loitp/model/Uiza.java

package vn.loitp.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class Uiza {

    @SerializedName("domainApi")
    @Expose
    private String domainApi;
    @SerializedName("appId")
    @Expose
    private String appId;
    @SerializedName("token")
    @Expose
    private String token;
    @SerializedName("urlLinkPlayEnvironment")
    @Expose
    private String urlLinkPlayEnvironment;
    @SerializedName("metadataId")
    @Expose
    private String metadataId;

    public String getDomainApi() {
        return domainApi;
    }

    public void setDomainApi(String domainApi) {
        this.domainApi = domainApi;
    }

    public String getAppId() {
        return appId;
    }

    public void setAppId(String appId) {
        this.appId = appId;
    }

    public String getToken() {
        return token;
    }

    public void setToken(String token) {
        this.token = token;
    }

    public String getUrlLinkPlayEnvironment() {
        return urlLinkPlayEnvironment;
    }

    public void setUrlLinkPlayEnvironment(String urlLinkPlayEnvironment) {
        this.urlLinkPlayEnvironment = urlLinkPlayEnvironment;
    }

    public String getMetadataId() {
        return metadataId;
    }

    public void setMetadataId(String metadataId) {
        this.metadataId = metadataId;
    }

}

File: core/src/main/java/com/model/Uiza.kt
package com.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class Uiza {

    @SerializedName("domainApi")
    @Expose
    var domainApi: String? = null
    @SerializedName("appId")
    @Expose
    var appId: String? = null
    @SerializedName("token")
    @Expose
    var token: String? = null
    @SerializedName("urlLinkPlayEnvironment")
    @Expose
    var urlLinkPlayEnvironment: String? = null
    @SerializedName("metadataId")
    @Expose
    var metadataId: String? = null

}
